e2e_tests/database/test_database.json

```diff
@@ -1018,10 +1018,10 @@
     "accepted": false,
     "key": "bkneutb8zck96tjknok7hpwwybuif0a1pgny8txer0dqixbexryfntojnhzlddwk",
     "sent": null,
+    "inviter": null,
```

Comment:
These changes appear, because i did some changes and did not recreate the file in my previous PR. They are irrelevant, though. But it does not hurt to check them in now. Less diffs in the future.